tests/arch/arm/arm_interrupt/src/arm_interrupt.c

```diff
@@ -191,6 +191,13 @@ void arm_isr_handler(const void *args)
 {
 	ARG_UNUSED(args);
 
+#if defined(CPU_HAS_FPU) && defined(ARM)
+	/* Clear Floating Point Status and Control Register (FPSCR),
```

Comment:
add a sentence "...to prevent from having the interrupt line set to pending again, in case FPU IRQ is...."

Reply:
Done